web/src/components/molecules/Common/WorkspaceMenu/index.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  ApiOutlined,
+  HomeOutlined,
+  SearchOutlined,
+  SettingOutlined,
+  UsergroupAddOutlined,
+  UserSwitchOutlined,
+} from "@ant-design/icons";
+import { Menu } from "antd";
+import { ItemType } from "antd/lib/menu/hooks/useItems";
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+export interface Props {
+  inlineCollapsed: boolean;
+  isPersonalWorkspace?: boolean;
+  workspaceId?: string;
+  defaultSelectedKeys?: string[];
+}
+
+function getItem(
+  label: string,
+  key: string,
+  icon: any,
+  show: "personal" | "notPersonal" | "both"
+) {
+  return {
+    key,
+    icon,
+    label,
+    show,
+  };
+}
```

Comment:
This function doesn't really do anything. You can keep things simpler by just write out the objects themselves.